### PR TITLE
eliminate warnings for Swift 4.1

### PR DESCRIPTION
**Pull request by rpinz**
eliminate warnings for Swift 4.1

### Checklist

- [ ] Circle CI is passing (code compiles and passes tests).
- [ ] There are no breaking changes to public API.
- [ ] New test cases have been added where appropriate.
- [ ] All new code has been commented with doc blocks `///`.

### CODE REVIEW

**Comment by grundoon** [comment]
@rpinz Looks like these are for vapor2 branch, not master?

**Comment by rpinz** [comment]
Correct.  These eliminate warnings while compiling Vapor2 with Swift 4.1.

**Comment by grundoon** [comment]
Right, but look above: "rpinz wants to merge 1 commit into vapor:master from rpinz:warnings" – this merge is set to `master`

File: Sources/Vapor/HTTP/Accept.swift [hunk 1]
```diff
@@ -33,6 +33,27 @@ extension Request {
             return []
         }
 
+        #if swift(>=4.1)
+        let accepts: [Accept] = acceptString.toCharacterSequence().split(separator: ",").compactMap { acceptSlice in
```
Comment (LoganWright) [inline Sources/Vapor/HTTP/Accept.swift]: I think we can reduce code duplication between this and 4.0 a bit somehow.

File: Sources/Vapor/HTTP/AcceptLanguage.swift [hunk 2]
```diff
@@ -11,6 +11,27 @@ extension Request {
             return []
         }
         
+        #if swift(>=4.1)
+        return acceptLanguageString.toCharacterSequence().split(separator: ",").compactMap { acceptLanguageSlice in
```
Comment (LoganWright) [inline Sources/Vapor/HTTP/AcceptLanguage.swift]: Same thing here, maybe we should break out some of the compactMap contents into a separate function or closure that we toss into the compactMap. What do you think?

**Comment by LoganWright** [comment]
Absorbed by other PR.

**Pull request closed without merging**